projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -143,6 +143,9 @@ function () {
 				return Jetpack::admin_url( array( 'page' => 'jetpack#/newsletter' ) );
 			}
 		);
+
+		// Track categories created through the category editor page
+		add_action( 'wp_ajax_add-tag', array( $this, 'track_newsletter_category_creation' ), 1 );
```

Comment:
Needed to prioritize the action order higher because when the actual function `wp_ajax_add_tag` executes it will send the response out without allowing actions hooked into it after to execute.